com.mojang.Minecraft.json

```diff
@@ -0,0 +1,65 @@
+{
+  /* GTK+ is needed by JavaFX. */
+  "app-id": "com.mojang.Minecraft",
+  "runtime": "org.freedesktop.Sdk.Extension.openjdk9",
+  "runtime-version": "1.6",
+  "sdk": "org.freedesktop.Sdk",
+  "command": "minecraft",
+  "finish-args": [
+      "--persist=.minecraft",
+      /* X11 access */
+      "--socket=x11",
+      /* OpenGL access */
+      "--device=dri",
+      /* PulseAudio access */
+      "--socket=pulseaudio",
+      /* Network access */
+      "--share=network",
+      "--extra-data=Minecraft.jar:e8d90b2637b141b556670d8460d317aadb6e4f2e644e519995c428800b25f212:280212:280212:http://s3.amazonaws.com/Minecraft.Download/launcher/Minecraft.jar"
```

Comment:
If you have a recent `flatpak` there is a new source type that cleans this up. https://github.com/flathub/com.discordapp.Discord/blob/master/com.discordapp.Discord.json#L33-L39